packages/typescript-configs/dom.json

```diff
@@ -5,12 +5,11 @@
     "lib": [
       "dom",
       "dom.iterable",
-      "es2017",
-      "esnext.asynciterable",
-      "scripthost"
+      "scripthost",
+      "es2020"
     ],
     "module": "esnext",
-    "target": "es2017"
+    "target": "es2020"
```

Comment:
ES2017 is still the best compilation target for modern browsers (supports 95% of user agents without doing anything), as it can be served without further transpiling using script + module attribute. 
Then, apps who really want to support the additional 5% of browsers can transpile that first target into ES5, and serve it using script + nomodule. 

What I'm trying to say is that by keeping the target to ES2017, we're removing the need for a transpiler such as babel in most cases.

Reply:
We don't want `tsc` as part of our build chain, babel (or in a potential future esbuild) for single file transpilation is much better suited to our needs as they allow finer grained transpiling based on browser (/node) support of single features rather than whole standard versions.

Ideally `tsc` should only be used for type-checking, and we use something else to to builds. Thus this target doesn't really do anything, I'm just setting it here to match the lib value.

Reply:
Thank you, that makes sense!